xds/src/test/java/io/grpc/xds/CdsLoadBalancerTest.java

```diff
@@ -325,6 +330,88 @@ public void handleCdsConfigs() throws Exception {
     assertThat(xdsClientRef.xdsClient).isNull();
   }
 
+  private static class FakeSocketAddress extends SocketAddress {
+    final String name;
+
+    FakeSocketAddress(String name) {
+      this.name = name;
+    }
+
+    @Override
+    public String toString() {
+      return "FakeSocketAddress-" + name;
```

Comment:
Where is this used? Why create `FakeSocketAddress` class?

Reply:
This is used on line 392. `CreateSubchannelArgs` has to have non-null `List<EquivalentAddressGroup> addrs` and the `EquivalentAddressGroup` has to have a non-null `SocketAddress` :-( 

I'll be happy to get rid of this if there is a better way to test `createSubchannel`.

Reply:
Is it possible to simply use `new InetSocketAddress("foo.example.com", 8080)`?

Reply:
Sure. `FakeSocketAddress` is defined and used quite a lot in various places in the gRPC test code, so I thought that's the more desirable pattern. May be we should have a single `FakeSocketAddress` defined that can be used in all such places.

Problem with `new InetSocketAddress("foo.example.com", 8080)` is an additional dependency on `java.net.InetSocketAddress` (not a big deal) and an unnecessary `getHostByName` call to resolve `foo.example.com` which I can avoid by using `InetAddress.anyLocalAddress()` instead of `foo.example.com` .